test/parallel/test-cluster-fork-windowsHide.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const cluster = require('cluster');
+
+if (cluster.isMaster) {
+  const msg = 'foobar';
+
+  cluster.setupMaster({
+    silent: true,
+    windowsHide: true
+  });
+
+  const worker = cluster.fork();
+  const cstdin = worker.process.stdin;
+  const cstdout = worker.process.stdout;
+  let response = '';
+
+  worker.on('exit', common.mustCall((code, signal) => {
+    assert.strictEqual(code, 0);
+    assert.strictEqual(signal, null);
+  }));
+
+  cstdout.on('data', (data) => {
+    response += data.toString('utf8');
+
+    if (response === msg) {
+      worker.disconnect();
+    }
+  });
+  cstdin.write(msg, 'utf8');
+} else {
+  process.stdin.on('data', (data) => {
+    process.stdout.write(data);
+  });
+  process.on('disconnect', function() {
```

Comment:
It doesn't really matter, just curious why you used a `function` here but arrow functions everywhere else.

Reply:
My bad. A little careless. I wasn't expecting the stdin would block the process from exiting at the beginning, and put the `disconnect` handler a little later, and not notice the coding style difference.